ONSdigital/alpha-eq-survey-runner
# Integrated first draft of survey response design

**What**

A first pass of integration of survey response designs, need to sort out thanks.html
next and mobile test this, but looking in a good state, ready for test.

**How to test**
1. Deploy a new environement using eq-terraform, editing the branch to be deployed to `digeq-80-admin-first-draft`
2. Check that a rendered questionnaire is presented correctly.

**Who can test**

Anyone but @dhilton
